deploy/cephfs/kubernetes/v1.13/csi-nodeplugin-rbac.yaml

```diff
@@ -3,6 +3,8 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
```

Comment:
there  is a PR https://github.com/ceph/ceph-csi/pull/764  to remove support  for v1.13 lets not update 1.13 related files